contributors/devel/sig-testing/monitoring.md

```diff
@@ -0,0 +1,264 @@
+ - [Triaged Flaky
+   Tests](https://github.com/kubernetes/kubernetes/issues?q=is%3Aissue+is%3Aopen+kind%2Fflake+-label%3Aneeds-triage+)
+ - [CI Signal board](https://github.com/orgs/kubernetes/projects/11) which
+   segments logged issues by workflow
+ 
+If an issue has already been opened for the test  you can add any new
+findings that are not already documented in the issue.
+
+For example, if a test is flaking intermittently and you have found another
+incident where the test has failed that has not been recorded in the issue then
+add the new information to the existing issue.
+
+You can:
+
+ - add a link to the Prow job where the latest test failure has occurred. 
+ - note the error message
+  
+New evidence is especially useful if the root cause of the problem with the test
+has not yet been determined and the issue still has a *needs-triage* label.
+
+If the issue has not already been logged please open up an issue on Kubernetes,
+https://github.com/kubernetes/kubernetes/issues/new/choose, and choose the
+appropriate issue template, [Failing Test ](https://github.com/kubernetes/kubernetes/issues/new?assignees=&labels=kind%2Ffailing-test&template=failing-test.md) or [Flaking
+Test](https://github.com/kubernetes/kubernetes/issues/new?assignees=&labels=kind%2Fflake&template=flaking-test.md)
+
+#### Filling out an issues
+Both test issue templates are reasonably self explanatory so what follows here
+are guidelines and tips on filling out the templates.
+
+When logging a Flaking or Failing test please
+ - use plain text when referring to test names, job names. Inconsistent
+   formatting of names makes it harder to process issues via automation.
+ - keep an eye out for test names that contain markdown parse-able formatting.
+   If you are a test maintainer refrain from including markdown in strings that
+   are used to name your tests, test components etc.
+ 
+#### Fill out the issue for a Flaking Test
+
+1. **Which jobs are flaking**
+
+The job is the tab in TestGrid that you are looking at.
+
+<img src="./testgrid-images/testgrid-jobs.png" height="50%" width="40%">
+
+The above example was taken from the SIG Release dashboard and we can see that
+* `conformance-ga-only` https://testgrid.k8s.io/sig-release-master-blocking#conformance-ga-only,
+* `skew-cluster-latest-kubectl-stable1-gce` https://testgrid.k8s.io/sig-release-master-blocking#skew-cluster-latest-kubectl-stable1-gce,
+* `gci-gce-ingress` https://testgrid.k8s.io/sig-release-master-blocking#gci-gce-ingress,
+* `kind-master-parallel` https://testgrid.k8s.io/sig-release-master-blocking#kind-master-parallel
+
+are flaky 
+
+2. **Which tests are flaking**
+
+Let's grab an example from the SIG release dashboards and look at the
+`node-kubelet-features-master` job in
+https://testgrid.k8s.io/sig-release-master-informing#node-kubelet-features-master.
+
+<img src="./testgrid-images/failed-tests.png" height="70%" width="100%">
```

Comment:
I made a slightly annotated version of this image so that we don't have to keep repeating "the value above that" or "the value below that" and it's a little easier for the "Since when has it been failing" section. 

![image](https://user-images.githubusercontent.com/13409103/106927840-8ed58f00-66e0-11eb-9bcf-4173295779d0.png)

Reply:
Very nice, thank you :) 
Will use this.